src/components/comment/CommentView.jsx

```diff
@@ -1,22 +1,50 @@
-import React from "react";
+import React, {useState} from "react";
 import TimeAgo from "javascript-time-ago";
 import en from "javascript-time-ago/locale/en";
 import Constants from "../../constants/Constants";
 
+const UNKNOWN_AUTHOR = "Unknown author";
+
 const CommentView = (props) => {
+    TimeAgo.addLocale(en);
+    const time = new TimeAgo('en-US');
+
+    const [show, setShow] = useState(false);
+
     const parseJsonComment = (value, constant) => {
```

Comment:
I dont understand what this method and it is very wird.
Why it has generic parameters (value, constant) which does not say anything and then you even pass array:
[Constants.HAS_AUTHOR]

And why it is called parseJsonComment if it does not have anything todo with comment. Moreover, i am quite sure we will have such functionality in some utility class. I would search for something like JsonldUtils ... and if we don't have, it belongs there.

Reply:
Yes, I just realized now... I really don't know why I did this... I wwas converting a string to Json and then converting Json to string.... I guess I was tired that day.

Reply:
And is it resolved now ? @LaChope